docs/api-reference/security.md

```diff
@@ -0,0 +1,139 @@
+<p>Packages:</p>
```

Comment:
Reference the new API group from https://github.com/gardener/gardener/blob/fffa1be2577b638eb9ae160fad6eaa8fa6c96218/docs/README.md#L75-L83 and from https://github.com/gardener/gardener/blob/58d136a7753df1160da613018f63b59693951df6/docs/api-reference/README.md#L1-L9.